src/components/DropzoneArea.js

```diff
@@ -361,7 +360,7 @@ class DropzoneArea extends React.PureComponent {
                     </Fragment>
                 }
 
-                {(showAlerts || (showAlertsOnError && snackbarVariant === 'error')) &&
+                {((typeof showAlerts === 'boolean' && showAlerts)  || (typeof showAlerts !== 'boolean' && showAlerts.includes(snackbarVariant))) &&
```

Comment:
A safer check would be 
```js
Array.isArray(showAlerts)
```
so to avoid crash when passing a type which does not have an `includes` method.

Reply:
You are right. Done!